src/app/home-page/home-logged-out/home.component.html

```diff
@@ -14,205 +14,293 @@
   ~    limitations under the License.
   -->
 
-<div class="header">
-  <div class="container">
-    <div fxLayout="column wrap" fxLayoutGap="40px" fxLayoutAlign="space-around" class="top-section">
-      <div class="text-center">
-        <h1 class="title">Create, Share, Use</h1>
-        <h2 class="title caption">Docker Tools and Workflows for the Sciences</h2>
-
-        <button
-          *ngIf="!(user$ | async); else loggedIn"
-          mat-flat-button
-          color="accent"
-          routerLink="/login"
-          class="big-button"
-          data-cy="register-button"
+<div fxLayout="column" fxLayoutGap="4rem">
+  <div fxLayout="column" class="title">
+    <div
+      class="container"
+      fxLayout="row"
+      fxLayout.lt-md="column"
+      fxLayoutAlign="space-between center"
+      fxLayoutAlign.lt-md="space-between stretch"
+      fxLayoutGap="5rem"
+    >
+      <div fxLayout="column" fxLayoutGap="3rem" fxLayoutAlign="space-around start" fxFlex="50" fxFlex.lt-md="100">
+        <h2>An app store for bioinformatics</h2>
+        <span
+          >Dockstore is a free and open source platform for sharing reusable and scalable analytical tools and workflows. It’s developed by
+          the
+          <a target="_blank" rel="noopener noreferrer" href="https://www.cancercollaboratory.org/" class="white-link"
+            >Cancer Genome Collaboratory</a
+          >
+          and used by the <a target="_blank" rel="noopener noreferrer" href="https://www.ga4gh.org/" class="white-link">GA4GH</a>.</span
         >
-          Register with <span class="fa fa-github"></span> or <span class="fa fa-google"></span>
-        </button>
-
-        <ng-template #loggedIn>
-          <button mat-flat-button color="accent" routerLink="/accounts" class="big-button">Welcome {{ (user$ | async).username }}</button>
-        </ng-template>
-
-        <button mat-flat-button routerLink="/search" class="m-2 big-button" data-cy="homepage-search-button">
-          Browse tools and workflows
-        </button>
-
-        <div class="search-holder">
+        <div fxLayout="row" fxLayout.lt-sm="column" fxLayoutAlign="start start" fxLayoutGap="2rem">
+          <a [routerLink]="['/about']">
+            <button class="btn">
+              <img src="../../../assets/images/home/icons-navigation-about-white.svg" alt="dockstore icon" />
```

Comment:
Will `../../../assets` work when deployed? If there are other cases of three double dots, then it will. Mentioning it just in case.

Reply:
Yes they all have 3 double dots